sycl/test-e2e/bindless_images/dx12_interop/read_write_unsampled.cpp

```diff
@@ -1,8 +1,8 @@
-// REQUIRES: cuda
+// REQUIRES: cuda || (windows && level_zero && gpu-intel-dg2)
 // REQUIRES: windows
```

Comment:
DirectX always requires Windows anyway.

```suggestion
// REQUIRES: cuda || (level_zero && gpu-intel-dg2)
// REQUIRES: windows
```